src/main/java/bio/terra/janitor/service/janitor/TrackedResourceService.java

```diff
@@ -185,4 +183,10 @@ private static TrackedResourceInfo createInfo(TrackedResourceAndLabels resourceA
         .expiration(OffsetDateTime.ofInstant(resource.expiration(), ZoneOffset.UTC))
         .labels(resourceAndLabels.labels());
   }
+
+  private void abandonResourcesAndLog(TrackedResource trackedResource) {
```

Comment:
nit: maybe just `abandonResources`. I don't think logging is important enough to include in the function name.